lib/mdi/Makefile.icc

```diff
@@ -0,0 +1,30 @@
+# *
+# *_________________________________________________________________________*
+# *      Fortran Library for Reactive Force Field                           *
+# *      DESCRIPTION: SEE READ-ME                                           *
+# *      FILE NAME: Makefile                                                *
+# *      CONTRIBUTING AUTHORS: Hansohl Cho(MIT), Aidan Thompson(SNL)        *
+# *                            and Greg Wagner(SNL)                         * 
+# *      CONTACT: hansohl@mit.edu, athompson@sandia.gov, gjwagne@sandia.gov *
+# *_________________________________________________________________________*/
+
+SHELL = /bin/sh
+
+# which file will be copied to Makefile.lammps
+
+EXTRAMAKE = Makefile.lammps.empty
+
+# ------ MAKE PROCEDURE ------
+
+lib: 	$(OBJ)
+	mkdir -p build
+	cd build; cmake -Dlibtype=STATIC -Dlanguage=C -D CMAKE_C_COMPILER=icc -D CMAKE_CXX_COMPILER=icc ../MDI_Library; make
+	cp build/MDI_Library/libmdi.a .
```

Comment:
rather than copying files, you should set symbolic links as it is done with other packages. see lib/voronoi or lib/smd